ci/github-script/README.md

```diff
@@ -0,0 +1,13 @@
+# GitHub specific CI scripts
+
+This folder contains `actions/github-script`-based JavaScript code.
+It provides a `nix-shell` environment to run and test these actions locally.
+
+To run any of the scripts locally:
+
+- Provide `gh` on `PATH` and make sure it's authenticated.
```

Comment:
Maybe we should provide `gh` in the shell... If it's a dependency.

The user will still need to setup their auth, but at least they won't have to separately _install_\* it if they don't want to.

\*Or separately enter an additional shell